src/main/java/website/frontrow/artificial/intelligence/ArtificialIntelligence.java

```diff
@@ -0,0 +1,177 @@
+package website.frontrow.artificial.intelligence;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import website.frontrow.level.Level;
+import website.frontrow.logger.Log;
+import website.frontrow.logger.Logable;
+import website.frontrow.board.Enemy;
+import website.frontrow.board.Mover;
+import website.frontrow.board.Player;
+import website.frontrow.board.Unit;
+import website.frontrow.game.GameConstants;
+
+/**
+ * Class for the movement of AI units.
+ */
+public class ArtificialIntelligence 
+		implements Logable
+{
+	
+	private Level level;
+	private ArrayList<Player> players;
+	private ArrayList<Unit> units;
+	private ArrayList<Enemy> enemies;
+	private ArrayList<Integer> movementsX;
+	private ArrayList<Integer> movementsY;
+	private Unit player;
+	
+	
+	/**
+	 * Create a new artificial intelligence.
+	 * @param level - input a level to derive enemies from.
+	 */
+	public ArtificialIntelligence(Level level)
+	{
+		this.level = level;
+		this.enemies = new ArrayList<Enemy>();
```

Comment:
You can remove the explicit type argument. It's unnecessary. Thus the new code is new ArrayList<>();